TestProject/TestProject/Entities/ImplementersPlugin/ProtocolEntity.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ProtocolsEntity.swift
+//  TestProject
+//
+//  Created by Muhammad Taifoor Saleem on 27/01/2022.
+//
+
+import Foundation
+
+//Refer to To-Do List DB Requisitions
+
+protocol ProtocolEntity
+{
+    //Get Permanent Lists
+    //Return: Dictionary [List] (nil on failure)
+    func getPermanentListTitles() -> [List]?
```

Comment:
Let's revamp this to make it async data request.

Reply:
Done